### PR TITLE
Add GFX overlay to UBL mesh edit

### PR DESCRIPTION
**Pull request by cdedwards**
Fixed misplaced HAS_TEMP_BED in dogm_bitmaps.h

rename _lcd_babystep_zoffset_overlay to _lcd_zoffset_overlay_gfx and move it out of BABYSTEP_ZPROBE_OFFSET to we can use it for other things. Add this function into UBL mesh edit screen.

update all Configuration.h to add ENABLE_MESH_EDIT_GFX_OVERLAY

### CODE REVIEW

**Comment by Roxy-3D** [comment]
What happens on the 20x4 Character LCD's ?

**Comment by cdedwards** [comment]
Don't have one to test out. Probably the same as with the BABYSTEP_ZPROBE_GFX_OVERLAY
guess I could add a check in sanitycheck.h for DOGLCD

**Comment by Roxy-3D** [comment]
> guess I could add a check in sanitycheck.h for DOGLCD

Yes please!

File: Marlin/src/inc/SanityCheck.h [hunk 1]
```diff
@@ -726,7 +726,9 @@ static_assert(1 >= 0
     static_assert(WITHIN(UBL_PROBE_PT_2_Y, MIN_PROBE_Y, MAX_PROBE_Y), "UBL_PROBE_PT_2_Y can't be reached by the Z probe.");
     static_assert(WITHIN(UBL_PROBE_PT_3_Y, MIN_PROBE_Y, MAX_PROBE_Y), "UBL_PROBE_PT_3_Y can't be reached by the Z probe.");
   #endif
-
+  #if ENABLED(ENABLE_MESH_EDIT_GFX_OVERLAY) && !ENABLED(DOGLCD)
```
Comment (thinkyhead) [inline Marlin/src/inc/SanityCheck.h:729]: `DISABLED` is also available.

**Comment by thinkyhead** [comment]
Mistakes were made. Lessons were learned.

1. This PR wasn't ready to merge and should have been cleaned up properly.
  The new option should be next to the option it relates to.
2. This PR was merged before a corresponding PR was made for the 1.1.x branch.
  **_Please don't do that._** Always make a PR for the other branch, and don't merge one alone.
  Always merge both at around the same time.

**Pull request merged**